layouts/index.html

```diff
@@ -244,8 +244,14 @@ <h2>⛓ Top Streaks, Consec Days Won</h2>
 
 {{ range $i, $wordle := $wordles.ByDate }}
   {{ $thisW = $wordle }}
+  {{ $isWin := eq ($thisW.Params.state.gameStatus | lower) "win" }}
```

Comment:
we love the intermediate variable assignment. no need to comment when a variable assignment is there instead